packages/experiments/src/components/Chiclet/ChicletCard.base.tsx

```diff
@@ -49,13 +48,7 @@ export class ChicletCardBase extends React.Component<IChicletCardProps, {}> {
         {preview ? ( // render custom preview
           React.cloneElement(preview, { className: css(preview.props.className, customPreviewStyling) })
         ) : (
-          <Image
-            width={PREVIEW_IMAGE_WIDTH}
-            height={PREVIEW_IMAGE_HEIGHT}
-            src={image}
-            role="presentation"
-            alt={imageAlt ? imageAlt : undefined}
-          />
+          <img width={'auto'} height={'100%'} src={image} alt={imageAlt ? imageAlt : undefined} />
```

Comment:
Is there a reason this image styling is different than the one in ChicletCard? Sorry if we already discussed this and I missed it!